dotcom-rendering/src/web/components/Carousel.tsx

```diff
@@ -353,7 +353,6 @@ export const CarouselCard: React.FC<CarouselCardProps> = ({
 	branding,
 }: CarouselCardProps) => (
 	<LI
-		stretch={true}
```

Comment:
So having this didn't do anything?

It might be worth double checking this on Firefox and friends - I vaguely recall there being some issues around this

Reply:
Prior to this PR, the conditional in the `<LI >` styles meant if you specified a `percentage` (which this element does) then stretch styles would not be applied.

I updated the LI element in this PR so that `percentage` and `stretch` properties can be applied at once (which is needed to make the dynamic/fast layout work) - doing this broke the carousel so I removed the prop!

Reply:
Thanks for explaining 👍